src/Core/CHANGELOG.md

```diff
@@ -2,13 +2,20 @@
 
 ## NOT RELEASED
 
+## 1.9.0
+
 ### Added
 
 - Changed case of object's properties to camelCase.
 - Added documentation in class's headers.
```

Comment:
I think one can say "class headers".

"class's headers" is the "headers owned by a class".

Reply:
sounds better thank you